pkg/tracker/disputeStatus.go

```diff
@@ -49,21 +57,21 @@ func (b *DisputeTracker) Exec(ctx context.Context) error {
 
 	instance, err := getter.NewTellorGetters(contractAddress, client)
 	if err != nil {
-		fmt.Println("instance Error, disputeStatus")
+		level.Error(b.logger).Log("msg", "Error getting master instance", "err", err)
```

Comment:
the `error` word is already added by the logger so no need to have it twice, The same in few other places. Also words like `problem`, `Could not` are also not necessary and when reading the logs are just noise. The logger itself formats errors in a very good an distinct format.
```suggestion
		level.Error(b.logger).Log("msg", "getting master instance", "err", err)
```